lib/sycamore/sycamore/query/compare.py

```diff
@@ -0,0 +1,66 @@
+from typing import Any, Set
```

Comment:
Why not just have this in logical_plan.py? Seems like we don't need a separate source file for this.

Reply:
Was trying to keep it separate since its an eval framework but I can move it